### PR TITLE
doc: known_issues: Add HID configurator issue on Linux

### PR DESCRIPTION
**Pull request by MarekPieta**
Change adds description of HID configurator issue under Linux.

Jira: NCSDK-8304

### CODE REVIEW

**Comment by MarekPieta** [comment]
Waiting for review from @pdunaj

File: doc/nrf/known_issues.rst [hunk 1]
```diff
@@ -318,6 +318,14 @@ DESK-978: Directed advertising issues with SoftDevice Link Layer
 
   **Workaround:** Directed advertising is disabled by default for nRF Desktop.
 
+.. rst-class:: v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
+
+NCSDK-8304: HID configurator issues of peripherals connected over Bluetooth LE to Linux host
```
Comment (greg-fer) [inline doc/nrf/known_issues.rst:323]:
```suggestion
NCSDK-8304: HID configurator issues for peripherals connected over Bluetooth LE to Linux host
```

File: doc/nrf/known_issues.rst [hunk 2]
```diff
@@ -318,6 +318,14 @@ DESK-978: Directed advertising issues with SoftDevice Link Layer
...
+  Using :ref:`nrf_desktop_config_channel_script` for peripherals connected to host directly over Bluetooth LE may result in receiving improper HID feature report ID.
+  In that case, the device provides HID input reports, but it cannot be configured via HID configurator.
```
Comment (greg-fer) [inline doc/nrf/known_issues.rst:325]:
```suggestion
  In such case, the device will provide HID input reports, but it cannot be configured with the HID configurator.
```

File: doc/nrf/known_issues.rst [hunk 3]
```diff
@@ -318,6 +318,14 @@ DESK-978: Directed advertising issues with SoftDevice Link Layer
...
+
+  **Workaround:** Connect the nRF Desktop peripheral through USB or via nRF Desktop dongle.
```
Comment (greg-fer) [inline doc/nrf/known_issues.rst:327]:
```suggestion
  **Workaround:** Connect the nRF Desktop peripheral through USB or using the nRF Desktop dongle.
```

**Review by greg-fer — CHANGES_REQUESTED** [review]
Couple of edits.

**Comment by MarekPieta** [comment]
I squashed the changes

**Review by umapraseeda — APPROVED** [review]

**Review by greg-fer — APPROVED** [review]

**Review by pdunaj — APPROVED** [review]

**Pull request merged**